libnvme/meson.build

```diff
@@ -46,7 +46,7 @@ if have_python_support
     configure_file(
         input:  '__init__.py', 
         output: '__init__.py',
-        copy: true,
+        configuration: conf,
```

Comment:
This should be in a separate patch. And even so, this would update the minimum requirement of meson to 0.49.0. I am *very* hesitant to do this. This needs to be properly documented why this is really necessary.

Reply:
Why does it bump the minimum requirement?

Reply:
I read the documentation incorrect. The support for dictionaries is depending on 0.49.0 but `conf` is a `configuration_data` object.